File: Android/GoodBam/app/src/main/java/com/example/goodbam/MainActivity.java
package com.example.goodbam;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageButton;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    private ImageButton btn_chart, btn_setting, btn_nightmode;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btn_chart = findViewById(R.id.main_btn_chart);
        btn_setting = findViewById(R.id.main_btn_setting);
        btn_nightmode = findViewById(R.id.main_btn_nightmode);

        btn_chart.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Toast.makeText(getApplicationContext(), "아직 미구현", Toast.LENGTH_SHORT).show();
                Intent intent = new Intent(MainActivity.this, ChartPageActivity.class);
                startActivity(intent);
            }
        });
        btn_setting.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Toast.makeText(getApplicationContext(), "아직 미구현", Toast.LENGTH_SHORT).show();
            }
        });
        btn_nightmode.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Toast.makeText(getApplicationContext(), "아직 미구현", Toast.LENGTH_SHORT).show();
            }
        });

    }
}

File: Android/GoodBam/app/src/main/java/com/example/goodbam/MainActivity.kt
package com.example.goodbam

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler
import android.os.HandlerThread
import kotlinx.android.synthetic.main.activity_start.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {

        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_start)

//        button.setOnClickListener{
//            val intent = Intent(this,LoginPageActivity::class.java)
//            startActivity(intent)
//            overridePendingTransition(R.anim.fadein,R.anim.fadeout)
//        }
        val intent = Intent(this,LoginPageActivity::class.java)

        val handler = Handler()
        handler.postDelayed({
            startActivity(intent)}
            , 3000L)

        handler.postDelayed({
            overridePendingTransition(R.anim.fadein,R.anim.fadeout)}
            , 3000L)

    }
}